### PR TITLE
renamed acore and dacore token names to mcore and dmcore

### PR DESCRIPTION
**Pull request by miladz68**
I Think this PR should be merged only after @alinetskyi has made preparations for a devnet restart.

- - -
This change is [Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)

### CODE REVIEW

**Review by wojtek-coreum — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: all files reviewed, 1 unresolved discussion (waiting on @miladz68, @oliverasistemas, @xlab, and @ysv)

*[a discussion](https://reviewable.io/reviews/coreumfoundation/coreum/127#-N8wc1HW0DPwUeJkDrkc:-N8wc1HX4cLaw3B0c2QL:b-ltvvly) (no related file):*
It must also be adjusted inside `CustomZNetNetworkConfig`. It means it would require another hack to update both repos.
In one of next PRs I will move `CustomZNetNetworkConfig` to coreum because I hit similar issue. Let's wait until it's done.

___

**Review by ysv — COMMENTED** [review]
This will require significant amount of time
I would stick to keep development and have devnet updated later 

> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127#-:-N8x2xwd82jsHbdqsAcF:b-ghns0e)* status: all files reviewed, 1 unresolved discussion (waiting on @miladz68, @oliverasistemas, and @xlab)

**Review by ysv — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: all files reviewed, 1 unresolved discussion (waiting on @miladz68, @oliverasistemas, and @xlab)

**Review by wojtek-coreum — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @oliverasistemas and @xlab)

**Review by miladz68 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @oliverasistemas and @xlab)

*[a discussion](https://reviewable.io/reviews/coreumfoundation/coreum/127#-N8wc1HW0DPwUeJkDrkc:-N9CKk-BBwrt1nXzRXa1:b1ymcm1) (no related file):*
<details><summary><i>Previously, wojtek-coreum (Wojtek) wrote…</i></summary><blockquote>

It must also be adjusted inside `CustomZNetNetworkConfig`. It means it would require another hack to update both repos.
In one of next PRs I will move `CustomZNetNetworkConfig` to coreum because I hit similar issue. Let's wait until it's done.
</blockquote></details>

@wojtek-coreum you already did this. 

___

**Review by wojtek-coreum — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @oliverasistemas and @xlab)

File: app/network.go [hunk 1]
```diff
@@ -35,10 +35,10 @@ const (
 
 // Known TokenSymbols
 const (
-	// TODO (milad): rename TokenSymbol to acore or attocore
-	// naming is coming from https://en.wikipedia.org/wiki/Metric_prefix
-	TokenSymbolMain string = "core"
-	TokenSymbolDev  string = "dacore"
+	// m prefix stands for milli, more info here https://en.wikipedia.org/wiki/Metric_prefix
+	TokenSymbolMain string = "mcore"
```
Comment (dzmitryhil) [inline app/network.go:39]: Do we register those token in the bank denoms metadata?

**Review by dzmitryhil — DISMISSED** [review]

**Review by ysv — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: all files reviewed, 1 unresolved discussion (waiting on @dzmitryhil and @oliverasistemas)

___
*[`app/network.go` line 39 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/127#-N8wVYZEkKHKwvWzjPZC-r1-39:-NBI8qcg0OJh8HZ8SdD1:bcztfn4) ([raw file](https://github.com/coreumfoundation/coreum/blob/03c478d255f92dc0727d260b3e3dd67295b0592d/app/network.go#L39)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

Do we register those token in the bank denoms metadata?
</blockquote></details>

pls recheck all places once more because we had plenty of updates after this

**Review by miladz68 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: all files reviewed, 1 unresolved discussion (waiting on @dzmitryhil and @oliverasistemas)

___
*[`app/network.go` line 39 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/127#-N8wVYZEkKHKwvWzjPZC-r1-39:-NBMFVHB7aOWPyt1ldt5:b-3tahw6) ([raw file](https://github.com/coreumfoundation/coreum/blob/03c478d255f92dc0727d260b3e3dd67295b0592d/app/network.go#L39)):*
<details><summary><i>Previously, ysv (Yaroslav Savchuk) wrote…</i></summary><blockquote>

pls recheck all places once more because we had plenty of updates after this
</blockquote></details>

yes, we do. we use these values to generate the genesis file.

**Review by dzmitryhil — COMMENTED** [review]
> Reviewed 1 of 28 files at r9, 40 of 40 files at r11, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @oliverasistemas)

**Review by ysv — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @oliverasistemas)

___
*[`app/network.go` line 39 at r1](https://reviewable.io/reviews/coreumfoundation/coreum/127#-N8wVYZEkKHKwvWzjPZC-r1-39:-NBMTSmG27GK5MHx0e80:b32wtd) ([raw file](https://github.com/coreumfoundation/coreum/blob/03c478d255f92dc0727d260b3e3dd67295b0592d/app/network.go#L39)):*
<details><summary><i>Previously, miladz68 (milad) wrote…</i></summary><blockquote>

yes, we do. we use these values to generate the genesis file. 
</blockquote></details>

ok

**Review by wojtek-coreum — APPROVED** [review]
> Reviewed 1 of 28 files at r9, 40 of 40 files at r11, 22 of 22 files at r12, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/coreumfoundation/coreum/127)* status: all files reviewed, 1 unresolved discussion (waiting on @miladz68 and @oliverasistemas)

___
*[`.github/workflows/ci.yml` line 25 at r12](https://reviewable.io/reviews/coreumfoundation/coreum/127#-NBMUqB75zA2Tj28qphD:-NBMUqB8EBgTU37A5_5-:b-n0pceh) ([raw file](https://github.com/coreumfoundation/coreum/blob/4ec810d1914386b66eae45a46263489e9d0787b6/.github/workflows/ci.yml#L25)):*
> ```YAML
>       - name: Lint, run unit tests and build
>         run: crust lint test build --log-format=yaml
>       # - name: Execute integration tests
> ```

Non-blocking: Keep in mind that this commit will cause all the PRs in all the repos to fail CI until crust is updated, so please be fast.
And in general I'm against this kind of hacks but I understand the urgent need.

**Pull request closed without merging**